controller/cmd/destination/main.go

```diff
@@ -20,6 +22,7 @@ func main() {
 	kubeConfigPath := flag.String("kubeconfig", "", "path to kube config")
 	k8sDNSZone := flag.String("kubernetes-dns-zone", "", "The DNS suffix for the local Kubernetes zone.")
 	enableH2Upgrade := flag.Bool("enable-h2-upgrade", true, "Enable transparently upgraded HTTP2 connections among pods in the service mesh")
+	disableIdentity := flag.Bool("disable-identity", false, "Disable identity configuration")
```

Comment:
I think if we wanted to disable identity globally, it would be best to avoid adding the controller at all. I guess this would be valuable if you wanted to opt-in to identity but have it off by default? It would be my preference that we don't make particularly easy to disable identity: it weakens the security from the default. I view this setting more as an escape-hatch for debugging and diagnostics.

Reply:
Specifically, I didn't want to make it impossible to start the destination service without an existing configuration. Maybe that's not worth optimizing for, though.

Reply:
Sorry for the delay here -- your reasoning all makes sense to me, and I'm good with next exposing this setting via the CLI.